app/app.rb

```diff
@@ -61,5 +61,8 @@ class App < Padrino::Application
     #     render 'errors/500'
     #   end
     #
+    get '/' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.